resources/service_broker.go

```diff
@@ -402,27 +407,34 @@ func (c CfServiceBrokerResource) diffServicesAccess(client cf_client.Client, ser
 	toDelete = make([]ServiceAccess, 0)
 	toCreate = make([]ServiceAccess, 0)
 
+	servicesAccessMapSrc := c.makeServicesAccessMap(servicesAccessSrc)
+	servicesAccessMapDest := c.makeServicesAccessMap(servicesAccessDest)
+
 	for _, serviceAccessSrc := range servicesAccessSrc {
-		if !c.containsServiceAccess(servicesAccessDest, serviceAccessSrc) {
+		exists := c.containsServiceAccess(servicesAccessMapDest, serviceAccessSrc)
+		if (!exists && !serviceAccessSrc.Disable) ||
```

Comment:
the condition just should be `if !exists || serviceAccessSrc.Disable`, am I missing something ?